AstroPhotometry/View/FileTree.xaml.cs

```diff
@@ -43,6 +43,12 @@ private void FileSystemObject_BeforeExplore(object sender, System.EventArgs e)
 
         private void InitializeFileSystemObjects()
         {
+            
+            var desktop_path = new DirectoryInfo(Environment.GetFolderPath(Environment.SpecialFolder.Desktop));
+            var desktop = new FileSystemObjectInfo(desktop_path);
+            //desktop.IsExpanded = true;
```

Comment:
Necessary?

Reply:
Yes, that's what put the desktop folder first

Reply:
Yes, that's what put the desktop folder first